src/research-department/dtos/department.find.dto.ts

```diff
@@ -28,6 +23,10 @@ export class ResearchDepartmentFindDto {
   @Min(1)
   @ExposeType(Number)
   limit?: number;
+  @IsOptional()
+  @IsEnum(ResearchDepartmentRelations, { each: true })
+  @Expose()
+  relations?: ResearchDepartmentRelations[];
```

Comment:
Like with researchDepartments

Reply:
It would be amazing if this could be supported for `FacilitiesFindDto` as well

Reply:
@alereca  common method was created under Utils to handle this 'relations' query string in all entities (Institution, Facility, ResearchDepartment). 
We can now use them as per our requirements in each API call with additional parameter ?relations